about.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="assets/styles/all.css">
+    <link rel="stylesheet" href="assets/styles/mobile-styles.css">
+    <link rel="stylesheet" href="assets/styles/desktop-styles.css">
+
+    <title>About Us</title>
+
+
+</head>
+
+<body>
```

Comment:
- [ ] I can see that you left some useless new lines that make your code less readable, Please remove all blank lines to make your code more readable. you can check the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)